subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -768,7 +768,7 @@ int bt_bap_broadcast_assistant_add_src(struct bt_conn *conn,
 
 		subgroup = net_buf_simple_add(&cp_buf, subgroup_size);
 
-		subgroup->bis_sync = param->subgroups[i].bis_sync;
+		subgroup->bis_sync = param->subgroups[i].bis_sync >> 1;
```

Comment:
Can you add a small comment for these as well as to why we are shifting, similar to what you did for the scan delegator?